lang/es.json

```diff
@@ -0,0 +1,28 @@
+{
+	"I18N.LANGUAGE": "Español",
+	"I18N.MAINTAINERS": "@Viriato139ac#0342",
```

Comment:
Small thing, but can you please replace the tabs at the beginning of these lines with four spaces so it is consistent with the rest of the lines?

Reply:
Done!